.github/workflows/build_and_test.yaml

```diff
@@ -172,11 +175,14 @@ jobs:
             diff test/kubelogo.png /tmp/5xx/secret-kubelogo.png &&
             echo -n "This absolutely exists" | diff - /tmp/5xx/absolute.txt &&
             echo -n "This relatively exists" | diff - /tmp/5xx/relative.txt &&
-            echo -n "500" | diff - /tmp/5xx/500.txt && 
+            echo -n "This new-dir exists" | diff - /tmp/5xx/change-dir.txt &&
+            echo -n "500" | diff - /tmp/5xx/500.txt &&
             ls /tmp/5xx/script_result &&
             kubectl exec sidecar -- sh -c "ls /tmp/" &&
             kubectl exec sidecar -- sh -c "! test -e /tmp/hello.world" && kubectl exec sidecar -- sh -c "test -e /tmp/change-hello.world" &&
             kubectl exec sidecar -- sh -c "! test -e /tmp/cm-kubelogo.png" && kubectl exec sidecar -- sh -c "test -e  /tmp/change-cm-kubelogo.png" &&
             kubectl exec sidecar -- sh -c "! test -e /tmp/secret-kubelogo.png" && kubectl exec sidecar -- sh -c "test -e /tmp/change-secret-kubelogo.png" &&
             kubectl exec sidecar -- sh -c "! test -e /tmp/absolute/absolute.txt" && kubectl exec sidecar -- sh -c "test -e /tmp/absolute/change-absolute.txt" &&
             kubectl exec sidecar -- sh -c "! test -e /tmp/relative/relative.txt" && kubectl exec sidecar -- sh -c "test -e /tmp/relative/change-relative.txt"
+            kubectl exec sidecar -- sh -c "! test -e /tmp/relative/relative.txt" && kubectl exec sidecar -- sh -c "test -e /tmp/relative/change-relative.txt"
```

Comment:
you need to concatenate all assertions with `&&` into a huge statement (or refactor to use some better assertion mechanisms...) otherwise failures in previous lines won't be relevant anymore to the step result.

Reply:
i have fixed the tests now. should be OK.

Reply:
Unfortunately this now fails when trying to deploy the kind k8s cluster (I suspect a github change on the build agents). I'll try to fix this in another branch...

Reply:
please rebase on master, then CI should be ok again